apps/trip_plan/lib/trip_plan/api/open_trip_planner/parser.ex

```diff
@@ -161,12 +161,14 @@ defmodule TripPlan.Api.OpenTripPlanner.Parser do
 
   defp parse_absolute_direction(nil), do: nil
 
-  defp id_after_colon(agency_colon_id) do
-    [agency, id] = String.split(agency_colon_id, ":", parts: 2)
-    # agency is either 1 (MBTA) or 2 (Massport)
-    case agency do
-      "1" -> id
-      "2" <> _ -> "Massport-" <> id
+  defp id_after_colon(feed_colon_id) do
+    [feed, id] = String.split(feed_colon_id, ":", parts: 2)
+
+    # feed id is either mbta-ma-us (MBTA) or 22722274 (Massport), if it's neither, assume it's MBTA
+    case feed do
+      "mbta-ma-us" -> id
+      "22722274" -> "Massport-" <> id
+      _ -> id
```

Comment:
question: If this is merged before the gtfs change, will this improperly label Massport ids as MBTA ids?

Reply:
suggestion: you could include `2` as a Massport ID, which should work with the existing feed?

Reply:
The current massport ID is `22722274`, we're not making any changes to that. The case statement is doing `"2" <> _`, so it's matching since it starts with 2. Based on the PR history, I think it used to be just an auto-generated 2, then we switched to their feed, which was using `22722274` explicitly.